src/cloud/docker/docker-extend.md

```diff
@@ -27,85 +27,129 @@ You can use the built-in extension mechanism of Docker to specify [multiple comp
 
 1. Add the `docker-entrypoint.sh` and `healthcheck` files if needed.
 
 1. Add any necessary `.conf` and `.ini` files for the service to the `etc` directory.
 
-1. Run the following command to build the image.
+1. Use the following command to build the image.
 
-   `docker build -t test/<service-name>:<service-version>`
+   ```bash
+   docker build -t test/<service-name>:<service-version>
+   ```
 
-1. Once the build succeeds, test the changes by specifying the [Docker build sources].
+1. After the build succeeds, test the changes by specifying the [Docker build sources].
 
 ## Add a new PHP extension
 
-In addition to built-in extensions, you can add PHP extensions to the PHP container.
+You can add PHP extensions to the PHP container by adding the extension configuration to the [ExtensionResolver.php] configuration file for {{ site.data.var.mcd-prod }}.
 
 {:.procedure}
 To add a new PHP extension:
 
-1. Clone the `{{site.data.var.mcd}}` project to your local environment if necessary.
+1. Clone the `{{site.data.var.mcd-package}}` project to your local environment.
 
 1. On the command line, navigate to the PHP directory.
 
    ```bash
    cd magento-cloud-docker/src/Compose/Php
+   ```
 
-1. Open the `ExtensionResolver.php` file, define the required extension inside the `getConfig` method by specifying the extension type and dependency.
+1. Add one or more extensions to the `ExtensionResolver.php` file:
 
-   For instance the following block adds the GNUPG extension:
+   -  Open the `ExtensionResolver.php` file for editing.
 
-   ```php?start_inline=1
-   'gnupg' => [
-     '>=7.0' => [
-     self::EXTENSION_TYPE => self::EXTENSION_TYPE_PECL,
-     self::EXTENSION_OS_DEPENDENCIES => ['libgpgme11-dev'],
-   ],],
-   ```
+   -  Specify the required extension in the `getConfig` method by specifying the extension type and dependency.
 
-   The extension type can be either CORE or PECL, which are defined as `EXTENSION_TYPE_PECL` or `EXTENSION_TYPE_CORE` respectively.
+      For example, the following block adds the `bcmath` extension:
+
+      ```php?start_inline=1
+      public static function getConfig(): array
+      ...
+
+      'bcmath' => [
+          '>=7.0' => [self::EXTENSION_TYPE => self::EXTENSION_TYPE_CORE],
+       ],
+       ...
+      ```
+      {:.no-copy}
+
+      In this case, the `bcmath` PHP core extension installs from `docker-php-source` images.
+
+      {:.bs-callout-info}
+      The configuration you specify depends on the location of the extension source files and installation method. You can add PHP core extensions from the official Docker PHP images, from the PECL repository, or using an installation script.  For details on the configuration attributes and format for the `getConfig` method, see [PHP extension configuration reference](#php-extension-configuration-reference).
```

Comment:
I think the first sentence, when distilled, may be "...depends on the location(x) and the method(x)".  Right? So, if that is true, then:

```suggestion
      The configuration you specify depends on the location of the extension source files and method of installation. You can add PHP core extensions from the official Docker PHP images, from the PECL repository, or using an installation script. For details on the configuration attributes and format for the `getConfig` method, see [PHP extension configuration reference](#php-extension-configuration-reference).
```